### PR TITLE
Remove Pine Forest and disable Towers and Lava Puddles

### PR DESCRIPTION
**Pull request by olliy1or**

### CODE REVIEW

**Comment by moreavy** [comment]
Are these the map voting results?

**Review by moreavy — APPROVED** [review]
I always support the map voting results.

**Comment by JostP** [comment]
> I always support the map voting results.

One of map voting result is that you will have to improve lava puddles map because it got average vote less than 2.7

**Comment by moreavy** [comment]
> > I always support the map voting results.
> 
> One of map voting result is that you will have to improve lava puddles map because it got average vote less than 2.7

Ah okay. I am working on it. I'm planning to change its name to `Volcanic Battlefield`.

**Comment by JostP** [comment]
Lava Puddles map will be disabled temporary until it is updated.

File: towers/map.conf [hunk 1]
```diff
@@ -44,3 +44,5 @@ chests.7.to = (-100,-24,-1)
 chests.8.n = 10
 chests.8.from = (35,-24,-100)
 chests.8.to = (100,-24,-1)
+
+disabled = true
```
Comment (JostP) [inline towers/map.conf:48]: It would be better if you added `disabled = true` to first line

Comment (JostP) [reply towers/map.conf:48]: So when you open file, you immediately see that it is disabled

**Review by JostP — APPROVED** [review]

**Pull request merged**